build-tools/packages/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts

```diff
@@ -665,12 +666,22 @@ function addPrettier(json: Record<string, any>) {
         "prettier:fix",
     );
 
-    const hasFormatScriptResolver = Object.prototype.hasOwnProperty.call(json.scripts, "format");
+    if (hasFormatScriptResolver || hasPrettierScriptResolver || hasPrettierFixScriptResolver) {
```

Comment:
@tylerbutler 

This is the if clause which executes resolver if and only if one of the scripts is in the `package.json`.